Source/Yingworld/Yingworld/Yingworld.csproj

```diff
@@ -33,21 +33,21 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="AlienRace">
-      <HintPath>..\..\..\..\..\..\..\workshop\content\294100\839005762\Assemblies\AlienRace.dll</HintPath>
+      <HintPath>..\..\..\..\AlienRaces\Assemblies\AlienRace.dll</HintPath>
```

Comment:
This may be questionable as it requires the repository to be cloned separately to installing the mod via Steam - however the Steam approach is not platform-agnostic, and really if someone is doing development work they should be able to build the dep assembly themselves.